source/docs/hardware-reference/talonfx/index.rst

```diff
@@ -17,6 +17,22 @@ The Falcon 500 powered by Talon FX is a brushless motor with an integrated motor
 
       Wiring and mount instructions in PDF format.
 
+Actuator Limits
+---------------
+
+CTR-Electronics actuators, such as the TalonFX, support various kinds of hardware and software limits.
+
+Documentation on retrieving and configuring limits can be found :doc:`here </docs/api-reference/api-usage/actuator-limits>`.
+
+Limit Switches
+^^^^^^^^^^^^^^
+
+CTR-Electronics supported actuators have limit features that will automatically neutral the actuator output (set voltage to 0) if a limit switch is activated. By default, limits are set to "normally open". This means that the switch needs to be explicitly closed (or grounded) for the actuator output to be set to neutral.
+
+When the limit switch is closed (connected to ground), the actuator will disable and blink both LEDs red in the direction of the fault (red blink pattern will move towards the M+/white wire for positive limit fault, and towards M-/green wire for negative limit fault).
```

Comment:
`(red blink pattern will move towards the M+/white wire for positive limit fault, and towards M-/green wire for negative limit fault)` doesn't make sense for TalonFX

Reply:
What is the replacement? I copied this from Phoenix 5 docs.

Reply:
The pattern will move toward the forward/reverse limit pin on the connector. So if it's limited forward it will strobe to the right (as oriented by the TalonFX text)

Reply:
Updated.